plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/launcher/PipelineArgumentsTab.java

```diff
@@ -459,7 +463,7 @@ private boolean validateRequiredProperties(MissingRequiredProperties validationF
     PipelineOptionsProperty missingProperty =
         Iterables.getFirst(validationFailures.getMissingProperties(), null);
     if (missingProperty != null) {
-      setErrorMessage("Missing required property " + missingProperty.getName());
+      setErrorMessage("Missing required property " + missingProperty.getName()); //$NON-NLS-1$
```

Comment:
This looks like a user-oriented message that should be externalized and parameterized.